File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    //for each row
    for (int i = 0; i < n; i++)
    {
        //for each column
        for (int k = n - 1; k > i; k--)
        {
            printf(" ");
        }
        for (int j = 0; j < i + 1; j++)
        {
            printf("#");
        }
        //print new line
        printf("\n");
    }
}

File: sentimental-hello/sentimental-mario-less/mario.py
from cs50 import get_int

while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

#count = 1
#for i in range(n):
#    print(' ' * (n-1),"#"*count)
#    count += 1
#    n -= 1

for i in range(n):
    for j in range(n):
        count =1
        print(' ' * (n-1), "#" * count)
        count += 1